### PR TITLE
Register nyholm/psr7 services for autowiring with HTTPlug factories

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | symfony/symfony-docs#...

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
@Nyholm maybe you want to approve this PR?

**Comment by Nyholm** [comment]
Thank you for the ping.

**Comment by Nyholm** [comment]
Thank you for the ping.

File: nyholm/psr7/1.0/config/packages/nyholm_psr7.yaml [hunk 1]
```diff
@@ -1,7 +1,4 @@
 services:
-    _defaults:
-        public: false
```
Comment (stof) [inline nyholm/psr7/1.0/config/packages/nyholm_psr7.yaml]: shouldn't it be kept for 3.4 ?

**Pull request merged**